### ISSUE TITLE
Mock ups by Sachito, check em out

### ISSUE THREAD

**Issue by smishra2** [routemaster/routemaster-web#6]
![Capture](https://f.cloud.github.com/assets/3464681/155511/18b6c69e-764e-11e2-8f5d-e03114c5ca38.JPG)
![Capture2](https://f.cloud.github.com/assets/3464681/155512/1a833ba6-764e-11e2-8d2b-09603a4d3cb5.JPG)
![Capture3](https://f.cloud.github.com/assets/3464681/155513/1c1c3b48-764e-11e2-89b8-f3d5d9802dd3.JPG)
![Capture4](https://f.cloud.github.com/assets/3464681/155514/1d7976a4-764e-11e2-9c5d-498d90e5e92c.JPG)
![Capture5](https://f.cloud.github.com/assets/3464681/155515/1f367c76-764e-11e2-9ccd-6a8dd36a822b.JPG)

**Comment by bgw** [comment]
I know much of this was already discussed IRL, but so that we have it somewhere in writing:
- The start and stop buttons should not be on the same screen
- There should be a path overview as a list
- Buttons with labels should line the top (or bottom) of the screens letting you jump between past paths, friend's paths, and achievements
- It it isn't too hard to implement, [horizontal swipes](https://www.youtube.com/watch?v=yDMN_3gIEII) should jump between the adjacent buttons
  - [Here's a demo app](http://cubiq.org/dropbox/SwipeView/demo/gallery/) that just does horizontal swipes between screens. The [project page is here](http://cubiq.org/swipeview).
  - To make the app faster and more fluid, we should try to [use partial page loads anyways](https://pjax.heroku.com/), so this might not be too hard to implement once we have that in place.
    - I'm not suggesting we use pjax specifically. It's probably best if we roll-our-own so that we can get a little more flexibility, and because our application (should be) really dynamic and javascripty. (Besides, partial page loads aren't that hard)
- There should also be "Log In with Google" and "Log In with OpenID" functionality in the application
